setup.py

```diff
@@ -112,17 +112,17 @@
         "SQLAlchemy>=1.4.35",
         "tqdm>=4.62.3,<5.0",
         "traitlets>=5.3.0",
-        "urllib3>=1.26.16",
+        "urllib3>=2.1.0,<3",
         "watchdog>=3.0,<4",
         # ** Dependencies maintained by Ethereum Foundation **
-        "eth-abi>=4.1.0,<5",
-        "eth-account>=0.8,<0.9",
-        "eth-typing>=3.4,<4",
-        "eth-utils>=2.2.0,<3",
+        "eth-abi>=4.2.1,<5",
+        "eth-account>=0.10.0,<0.11",
+        "eth-typing>=3.5.2,<4",
+        "eth-utils>=2.3.1,<3",
         "py-geth>=3.13.0,<4",
```

Comment:
NOTE: unable to upgrade to 4.0.0 until https://github.com/ethereum/py-geth/issues/167 resolved